### ISSUE TITLE
Style/RegexpLiteral false reports on %r/.../

### ISSUE THREAD

**Issue by zenspider** [bbatsov/rubocop#1613]
```ruby
%r/\A\xEF\xBB\xBF/
```

**Comment by rrosenblum** [comment]
@zenspider What is your configuration for `MaxSlashes`, and what is the message that RuboCop reports to you for this error?

I was testing this out with a the default value of `MaxSlashes: 1` and this appears to be working fine for me. The message that I get is `Use %r only for regular expressions matching more than 1 '/' character.`

**Comment by bbatsov** [comment]
We changed this cop recently, so this should no longer be a problem.

**Comment by zenspider** [comment]
Thanks!
